config/routes.rb

```diff
@@ -4,11 +4,9 @@
   end
 
   namespace :api, :defaults => {:format => 'json'}  do
-    namespace :v1 do
-      constraints name: /(?!:\/\/)([a-zA-Z0-9]+\.)?[a-zA-Z0-9][a-zA-Z0-9-]+\.[a-zA-Z]{2,9}?/ do
-        get 'domains/:name' => 'domains#show'
-        resources :domains, only: :create
-      end
+    constraints name: /(?!:\/\/)([a-zA-Z0-9]+\.)?[a-zA-Z0-9][a-zA-Z0-9-]+\.[a-zA-Z]{2,9}?/ do
+      get 'domains/:name' => 'domains#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.